tests/ci/cdk/util/build_spec_loader.py

```diff
@@ -0,0 +1,35 @@
+#!/usr/bin/env python3
+
+# Copyright Amazon.com, Inc. or its affiliates. All Rights Reserved.
+# SPDX-License-Identifier: Apache-2.0
+
+from aws_cdk import aws_codebuild as codebuild
+from util.metadata import TEAM_ACCOUNT, AWS_ACCOUNT, DEFAULT_REGION, AWS_REGION
+import yaml
+
+
+class BuildSpecLoader(object):
+    """Responsible for loading the BuildSpec yml file as python object."""
+
+    @staticmethod
+    def load(file_path):
+        """
+        Used to load yml file and replace some placeholders if needed.
+        :param file_path: path to the yml file.
+        :return: python object.
+        """
+        # If the deployment uses team account, the change of batch BuildSpec file is loaded automatically without deployment.
+        # else, the change will require manual deployment via CDK command.
+        if AWS_ACCOUNT == TEAM_ACCOUNT:
```

Comment:
Don't we want to verify that the region is the default region as well?

Reply:
Good catch. I will use `CAN_AUTOLOAD` to make this condition more clear.
```
# Indicate if the BuildSpec files can be automatically loaded without manualy deployment.
CAN_AUTOLOAD = (AWS_ACCOUNT == TEAM_ACCOUNT) and (AWS_REGION == DEFAULT_REGION)
```